app/src/main/java/cm/aptoide/pt/abtesting/experiments/SimilarAdExperiment.java

```diff
@@ -40,7 +41,7 @@ public SimilarAdExperiment(ABTestManager abTestManager, Scheduler scheduler,
           }
           switch (experimentAssigment) {
             case "appnext_ad":
-              return adsManager.loadAppnextAd(keywords);
+              return adsManager.loadAppnextAd(keywords, BuildConfig.APPNEXT_SIMILAR_PLACEMENT_ID);
```

Comment:
This method is called loadAppnextAd while the one from adRespository is called loadAppNextAd. Maybe change one of them to match the other